React/wbooks/src/redux/auth/actions.js

```diff
@@ -1,49 +1,45 @@
+import { completeTypes, createTypes, withPostSuccess } from 'redux-recompose';
+
 import api from '../../Config/api';
 import history from '../../Config/history';
 import localStorage from '../../Services/LocalStorage';
 import UserService from '../../Services/UserService';
 
-export const authActions = {
-  logout: 'LOGOUT',
-  login: 'LOGIN',
-  loginSuccess: 'LOGIN_SUCCESS',
-  loginFailure: 'LOGIN_FAILURE',
-  signup: 'SIGNUP',
-  signupSuccess: 'SIGNUP_SUCCESS',
-  signupFailure: 'SIGNUP_FAILURE'
-};
+export const actions = createTypes(completeTypes(['LOGOUT', 'LOGIN', 'SIGNUP']), '@@AUTH');
```

Comment:
Complete types are those that define a SUCCESS and FAILURE action, which you don't have in logout:
https://github.com/Wolox/redux-recompose/blob/master/src/completers/completeTypes/docs.md

completeTypes receives 2 parameters, the first is an array of thunk actions (those that have success or failure), and the second one is an array of those actions that don't have succes or failure (your logout, for example). So you should be doing the following:

```
export const actions = createTypes(completeTypes(['LOGIN', 'SIGNUP'], ['LOGOUT']), '@@AUTH');
```